powersimdata/input/const/grid_const.py

```diff
@@ -96,6 +96,7 @@
     "mu_Vmax",
     "mu_Vmin",
     "interconnect",
+    "eia_id",
```

Comment:
You also need to add `"int"`to the `col_type_bus` list right before the second to last element. Not sure, we actually use the type of the field now that that we don't translate the content **grid.mat** file to  a `Grid` object but still it would be cleaner